packages/mux-uploader/src/mux-uploader-drop.ts

```diff
@@ -1,12 +1,30 @@
 import { globalThis, document } from './polyfills';
+import { getMuxUploaderEl } from './utils/element-utils';
+import type MuxUploaderElement from './mux-uploader';
 
 const template = document.createElement('template');
 
-/** @todo: Currently removing all styles. Follow up on overlay styling (CJP) */
-template.innerHTML = `
+template.innerHTML = /*html*/ `
 <style>
   :host {
     position: relative;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border: 2px dashed #ccc;
+    padding: 2.5rem 2rem;
+    border-radius: .25rem;
+  }
+
+  #heading {
```

Comment:
question(non-blocking): if an element is slotted into the `heading` slot, this won't select any visible elements. Would that be an issue? Or is that OK since users likely would need/want to account for that styling?

Reply:
@gkatsev @luwes what do you think if we did this instead?

```css
  slot[name='heading'] > * {
    margin-bottom: 0.75rem;
    font-size: 1.75rem;
  }

  slot[name='separator'] > * {
    margin-bottom: 0.75rem;
  }
```